fe-vm/src/util/makePriceFormat.js

```diff
@@ -0,0 +1,3 @@
+export const makePriceFormat = (priceString) => {
+  return priceString ? Number(priceString).toLocaleString() + "원" : "";
+};
```

Comment:
파라미터에 default value를 주어서 삼항연산자를 없애볼 수 있겠네요~
```suggestion
export const makePriceFormat = (priceString = '') => {
  return Number(priceString).toLocaleString() + "원";
};
```